libparsec/crates/client/src/certificates_ops/add.rs

```diff
@@ -1079,6 +1080,10 @@ async fn check_realm_role_certificate_consistency(
         Some(user_certificate),
     )
     .await?;
+
+    // TODO: Something goes wrong there: Outsider should not be allowed to be
+    // MANAGER/OWNER of a shared realm (see share_realm_with_outsider test).
```

Comment:
A relevant issue as been opened ? I can't find it

Reply:
Opened: https://github.com/Scille/parsec-cloud/issues/5545